paella-matterhorn/plugins/edu.harvard.dce.infoButtonPlugin/dce_infobutton.js

```diff
@@ -14,18 +14,28 @@ Class ('paella.plugins.InfoPlugin', paella.ButtonPlugin,{
     var thisClass = this;
 
     var popUp = jQuery('<div id="dce-info-popup"></div>');
-    var buttonActions =[ 'Help with this player', 'Report a problem', 'Feedback', 'All Course Videos' ];
+    var buttonActions =[ 'Help with this player', 'Report a problem', 'DCE system status', 'Feedback', 'All Course Videos' ];
 
+    jQuery(popUp).append(thisClass.getItemTitle());
     buttonActions.forEach(function(item){
       jQuery(popUp).append(thisClass.getItemButton(item));
     });
     jQuery(domElement).append(popUp);
   },
 
+  getItemTitle: function () {
+    var mpInfo = paella.matterhorn.episode.mediapackage;
+    var titleDiv = mpInfo.title?'<span>' + mpInfo.title + '</span>':'';
+    var seriesTitleDiv = mpInfo.seriestitle?'<span>' + mpInfo.seriestitle + '</span>':'';
+    var elem = jQuery('<div />');
+    jQuery(elem).attr({'class': 'infoPubTitle'}).html(seriesTitleDiv + titleDiv);
```

Comment:
Jay, this is consistent with the upstream syntax for the version of UPV we're using.